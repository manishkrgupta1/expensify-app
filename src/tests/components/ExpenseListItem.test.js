import {shallow} from 'enzyme';
import ExpenseListItem from '../../components/ExpenseListItem';
import React from 'react';
import {expenses} from '../fixtures(dummy).js/fixtures';


test('should render Expense List Item',()=>{
    const wrapper=shallow(<ExpenseListItem {...expenses[0]}/>);
    expect(wrapper).toMatchSnapshot();
    console.log('happy me');
})