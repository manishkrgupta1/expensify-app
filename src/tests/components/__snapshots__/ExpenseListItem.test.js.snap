// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Expense List Item 1`] = `
<div>
  <Link
    to="/edit/1"
  >
    gum
  </Link>
  <p>
    $10.00
    -
    January 1st, 1970
  </p>
</div>
`;
